### PR TITLE
Feature#8 add login styles

### PR DESCRIPTION
**Pull request by M-Cordova-M**
# Description

What did you implemented/Why did you implemented this:

We added styles on the sign-in view.

## Screenshots

reloj 4

### CODE REVIEW

**Review by oscaregr — APPROVED** [review]
good

**Review by arthurchina92 — APPROVED** [review]

**Review by MaxRetana — APPROVED** [review]
Looks good

**Review by Reiyirak — APPROVED** [review]
Nice!

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -100,4 +100,23 @@ form button {
...
+}
+.login_title {
+  align-items: center;
+  font-size: 50px;
+  font-weight: bold;
+  justify-content: center;
+  margin-bottom: 50px;
+  margin-top: 50px;
+  text-align: center;
+}
```
Comment (juperameza) [inline app/assets/stylesheets/application.css:122]: Add a new line after this

**Review by juperameza — DISMISSED** [review]
Lgtm

**Review by dagp-ucol — APPROVED** [review]

**Review by juperameza — APPROVED** [review]
Lgtm

**Review by MarcoRodriguez23 — APPROVED** [review]
LGFM !

**Pull request merged**